src/bot.py

```diff
@@ -94,7 +94,14 @@ def int_conv(string: str) -> bool:
                 continue
         except AttributeError:  # raised if there is no parent comment
             continue
-
+            
+        # bal command
+        if re.match(r"!bal", comment.body): # as regex so we can add matches easily if needed
```

Comment:
Won't this conflict with line 92? Won't the bot continue if it doesn't match `!(nug|nugget|gold)`?

Reply:
If people do two commands in one, i.e. `!bal !nug 4` it will. I think we should the regex to have to equal the message, rather than just be contained inside. Otherwise all other commands, i.e. the leaderboard one will have that issue, and people shouldn't be able to run multiple commands at once anyway.

Reply:
This will conflict, the bot will reject the comment due to not being a nug comment before it gets processed. This should really be !nug bal as opposed to !bal for the reasons I said earlier